templates/events/_header.html.twig

```diff
@@ -26,7 +26,7 @@
                 L'événement à été annulé
             </div>
         {% elseif event.isCitizenInitiative or not event.full%}
-            <a class="btn btn--no-border btn--full" href="{{ path('app_event_attend', {uuid: event.uuid.toString, slug: event.slug}) }}">Je veux participer</a>
+            <a class="btn btn--no-border btn--full" href="{{ path('app_' ~ event.type ~ '_attend', {uuid: event.uuid.toString, slug: event.slug}) }}">Je veux participer</a>
```

Comment:
Does not working

"Unable to generate a URL for the named route "app_#{event.type}_attend" as such route does not exist."